pkg/cmd/cli/cmd/set/env.go

```diff
@@ -200,8 +202,23 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 			return nil
 		})
 		if !ok {
-			skipped++
-			continue
+			ok, err = f.UpdateObjectEnvironment(info.Object, func(vars *[]kapi.EnvVar) error {
```

Comment:
I guess I am fine with two separate methods. @smarterclayton is there any use for this upstream?